ocaml/xapi/at_least_once_more.ml

```diff
@@ -50,7 +58,7 @@ let again (x : manager) =
           Thread.create
             (fun () ->
               (* Always do the operation immediately: thread is only created when work needs doing *)
-              x.f () ;
+              log_and_ignore_exn (x.f ()) ;
```

Comment:
I understand the `x.f ()` will be evaluated by call-by-value before passing to `log_and_ignore_exn`. As a result, the exception from the evaluation would not be captured by `log_and_ignore_exn`.

Reply:
Oh that's wrong isn't it... I should add the unit arg to `log_and_ignore_exn`.

Reply:
Fixed it.